certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -1527,6 +1591,9 @@
             }
           }
         },
+        "x-ms-openai-data": {
```

Comment:
Why is openai set to false above but true here?

Reply:
I am disabling other custom actions in open AI and enabling only one so that the other 2 dont interfere when prompts are given in the AI plugin.  In testing I have see that other 2 custom actions are invoked giving incorrect results.